mts-web-app/mts/templates/about.html

```diff
@@ -27,23 +28,255 @@ <h2>ABOUT</h2>
+                    Digital
+                    Humanities program at the CUNY Graduate Center.
+                    Estefany is interested in Peer Mentorship, Digital Learning, and Education Technology. In her free
+                    time,
+                    Estefany loves to take her loved ones on hiking adventures and explore the natural world around her.
+                </p>
+                <p>Estefany's passion for exploring her Indigenous roots has sparked a deep curiosity about Indigenous
+                    peoples
+                    in the Americas. She joined the More Than Surviving team project, supporting the Visual Design,
+                    Outreach,
+                    and Social Media Strategy. She is excited to work with her team and to learn about the Wampanoag
+                    people.
+                </p>
+            </div>
+        </div>
     </div>
     <br>
-    <br>
+   
     <hr id="about-hr">
     <div>
         <h1 id="tools">TOOLS</h1>
+        <h3>MAP</h3>
+        <H4>MAPBOX GL JS</H4>
+        <p>The map of Wampanoag political activity and activists is powered by <a
+                href="https://docs.mapbox.com/mapbox-gl-js/guides/">Mapbox GL JS</a>, a JavaScript mapping library. The
+            map highlights activism, as well as key locations associated with activists, and allows viewers to make
+            connections between events and activists across various Wampanoag communities. The goal of the map is to
+            reinforce Wampanoag presence in their homeland.</p>
+        <br>
+
+        <h3>TIMELINE</h3>
+        <H4>TIMELINEJS</H4>
+        <p>The timeline of Wampanoag political activity is powered by <a
+                href="https://timeline.knightlab.com/">TimelineJS</a>, an open source timeline creation tool developed
+            by Northwestern University's Knight Lab. The timeline includes key events in Wampanoag activism from the
+            antebellum period, juxtaposed alongside related events in Massachusetts and United States history. The goal
+            of the timeline is to illustrate the progression and persistence of Wampanoag activism and help viewers
+            place these acts of resistence in the larger context of national events.</p>
+        <br>
+
+        <h3>DATABASE</h3>
+        <h4>POSTGRESQL | GEOJSON | PYTHON | GOOGLE SHEETS</h4>
+        <p>More Than Surviving is powered by a custom-built API that enables seamless integration between the site's
+            database and the data visualized in the interactive map and timeline. For accessibility and ease of future
+            updates, the site database can be updated through a Google Colab notebook that retrieves research data that
+            is put into Google Sheets. In the future, this will allow tribe members and other researchers to easily join
+            and expand the project.</p>
     </div>
 </div>
 {% endblock content %}
```

Comment:
@eszypulski Add a newline at end of file.

Reply:
Done, thanks!